<launch>
    <arg name="drone_number" default="1"/>
    <node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk_$(arg drone_number)" output="screen">
    <!-- node parameters -->
    <param name="serial_name"   type="string"   value="/dev/ttyTHS2"/>
    <param name="baud_rate"     type="int"      value="921600"/>
    <param name="app_id"        type="int"      value="1024795"/>
    <param name="app_version"   type="int"      value="1"/>
    <param name="align_time"    type="bool"     value="true"/>
    <param name="enc_key"       type="string"   value="a5f11a418f3dcdc39c4a1efb35a9dd4545e47b56cae9d98c988468277a9305dc"/>
    <param name="use_broadcast" type="bool"     value="false"/>
    <param name="no_RTK"        type="bool"     value="true"/>
    </node>
</launch>
